llvm/lib/Target/DirectX/DXILPrettyPrinter.cpp

```diff
@@ -12,13 +12,27 @@
 #include "llvm/ADT/StringRef.h"
 #include "llvm/IR/PassManager.h"
 #include "llvm/Pass.h"
+#include "llvm/Support/FormatVariadic.h"
 #include "llvm/Support/raw_ostream.h"
 
 using namespace llvm;
 
 static void prettyPrintResources(raw_ostream &OS,
                                  const dxil::Resources &MDResources) {
```

Comment:
What does the "MD" in "MDResources" mean?

Reply:
"Metadata". This is mostly a temporary thing while both DXILResourceAnalysis and DXILResourceMDAnalysis co-exist. The one with "MD" in the name will eventually be removed.